src/main/java/com/yelp/nrtsearch/server/search/sort/SortParser.java

```diff
@@ -39,6 +39,7 @@
 public class SortParser {
   public static final BiFunction<SortField, Object, CompositeFieldValue>
       DEFAULT_SORT_VALUE_EXTRACTOR = (sortField, value) -> getValueForSortField(sortField, value);
+  public static final String NULL_SORT_VALUE = "NULL_SORT_VALUE";
```

Comment:
Should we document this somewhere? Maybe under a section called "Reserved Values"?
Also should we make it more unlikely to be used, maybe wrap in %? So `%NULL_SORT_VALUE%`?